### ISSUE TITLE
Apply button is missing at "Rest alignment" step.

### ISSUE THREAD

**Issue by stpiton** [Mwni/blender-animation-retargeting#11]
Spent couple of hours trying to understand what I am doing wrong. Apply button is missing at "Rest alignment" step. It turns out its a bug in 2.80.74 version. Reopened project in 2.83.3 and everything is fine.
![Retargeting_bug2 80](https://user-images.githubusercontent.com/72409024/95114708-043e4c80-074d-11eb-89af-f4c6932ab491.PNG)
![Retargeting_bug2 83](https://user-images.githubusercontent.com/72409024/95114710-056f7980-074d-11eb-8ec2-af90b01c5f11.PNG)

**Comment by Mwni** [comment]
Shame you wasted hours on it. Was there any errors printed in the console?
Nevertheless, thanks for the report.

**Comment by stpiton** [comment]
Do you mean log located in "info" window? Here are l
[Log2.80.74.txt](https://github.com/Mwni/blender-animation-retargeting/files/5333052/Log2.80.74.txt)
[Log2.83.3.txt](https://github.com/Mwni/blender-animation-retargeting/files/5333053/Log2.83.3.txt)

ogs for both version. In 2.83.3 button apears, but does nothing. After clicking on it Ive got "No Rest Alingment" status

**Comment by stpiton** [comment]
I've been able to finish set up. Dont know what exactly was wrong, but after switch to 2.9 the problem gone away. And now I must say digging out worth spent time. Workflow is impressivly powerfull. Should we close issue?

**Comment by Mwni** [comment]
Glad it's working for you now. The issue was simply a certain icon graphic not being available in 2.80.xx and below. I don't think I will fix this, but add it to the readme. Take care.

**Comment by stpiton** [comment]
For those who will face this problem in the future. After clicking "Set up" in Rest Alignment at least one bone needs to be transformed (rotated, scaled or moved). Otherwise addon not will not finish the setup. This may be confusing if both rigs are in same rest pose from the start, but its not a big deal to fix that.

**Comment by paddywwoof** [comment]
I suspect everyone using this addon wastes a bit of time with this issue. @Mwni, rather than me making a branch and submitting a pull request, why not add an extra line in the Readme.me ```NB at least one bone needs to be transformed (rotated, scaled or moved) otherwise the addon not will not finish the setup!```

Apart from that I have found your system to be better than any other I have tried: well done and thank you for all your work. I even managed to re-target mixamo animations to rigify by choosing the appropriate `fk` bones and switching IK-FK as needed.
